packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md

```diff
@@ -0,0 +1,75 @@
+### 1: Initiate password recovery
+
+To begin the password recovery flow, the user clicks the **Forgot Password** button on the login page. Then they select **Email** as the authenticator to be used for password recovery on the subsequent page and click **Submit**. Okta then sends an email to their email address matching the Forgot Password template that was altered earlier.
+
+<div class="common-image-format">
+
+![Screenshot of email sent to user](/img/advanced-use-cases/custom-pwd-recovery-custom-email.png)
+
+</div>
+
+The Reset Password link in the email includes the `OTP` and `request.relayState` variables as query parameters back to the application. For example,
+
+`https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}` becomes `https://localhost:44314/magiclink/callback?otp=726009&state=1b34371af02dd31d2bc4c48a3607cd32`.
+
+### 2: Handle the otp and state parameters
```

Comment:
```suggestion
### 2: Handle the OTP and state parameters
```

Reply:
Left as is. This refers to the parameter called otp rather than the concept of OTP so I think it should stand